allauth/account/templatetags/account_tags.py

```diff
@@ -3,4 +3,4 @@
 warnings.warn("{% load account_tags %} is deprecated, use {% load account %}",
               DeprecationWarning)
 
-from account import *  # noqa
+from .account import *  # noqa
```

Comment:
True -- the `_tags` files have been deprecated since 2012 and can be simply removed by now..